src/components/ObsDetailsDefaultMode/SavedMatch/SavedMatch.tsx

```diff
@@ -0,0 +1,69 @@
+import { useNetInfo } from "@react-native-community/netinfo";
+import { matchCardClassBottom, matchCardClassTop } from "components/Match/Match";
+import MatchHeader from "components/Match/MatchHeader";
+import PhotosSection from "components/Match/PhotosSection";
+import LocationSection from "components/ObsDetailsDefaultMode/LocationSection/LocationSection";
+import MapSection from "components/ObsDetailsDefaultMode/MapSection/MapSection";
+import { Button, ScrollViewWrapper } from "components/SharedComponents";
+import { View } from "components/styledComponents";
+import _ from "lodash";
+import React from "react";
+import type { RealmObservation } from "realmModels/types";
+import { useTranslation } from "sharedHooks";
+
+import SavedMatchHeaderRight from "./SavedMatchHeaderRight";
+
+type Props = {
+  observation: RealmObservation,
+  navToTaxonDetails: ( ) => void,
+}
+
+const SavedMatch = ( {
+  observation,
+  navToTaxonDetails
+}: Props ) => {
+  const { t } = useTranslation( );
+  const { isConnected } = useNetInfo( );
+
+  const latitude = observation?.privateLatitude || observation?.latitude;
+  const { taxon } = observation;
+
+  return (
+    <ScrollViewWrapper>
+      <SavedMatchHeaderRight uuid={observation.uuid} />
+      <View className={matchCardClassTop}>
+        <MatchHeader hideObservationStatus topSuggestion={observation} />
+      </View>
+      <PhotosSection
+        representativePhoto={taxon?.representative_photo}
```

Comment:
`representative_photo` is not persisted to the DB and hence not available here for this taxon.